lib/feedjira/parser/rss_entry.rb

```diff
@@ -27,14 +27,20 @@ class RSSEntry
       element :issued, as: :published
       elements :category, as: :categories
 
-      element :guid, as: :entry_id
+      element :guid, as: :entry_id, class: Feedjira::Parser::GloballyUniqueIdentifier
       element :"dc:identifier", as: :dc_identifier
 
-      # rubocop:disable Naming/MemoizedInstanceVariableName
+      def entry_id
+        @entry_id && @entry_id.guid
+      end
+
+      def url
+        @url || (@entry_id && @entry_id.url)
+      end
+
       def id
-        @entry_id ||= @dc_identifier || @url
+        entry_id || @dc_identifier || @url
```

Comment:
These lines now belong to lib/feedjira/rss_entry_utilities.rb, so please merge (or rebase onto) master and apply the change to it.